indra/llcharacter/llik.h

```diff
@@ -55,22 +57,27 @@ namespace LLIK
 class Solver;
 class Joint;
 
-// local flags
-constexpr U8 FLAG_LOCAL_POS = 1 << 0;
-constexpr U8 FLAG_LOCAL_ROT = 1 << 1;
-constexpr U8 FLAG_LOCAL_SCALE = 1 << 2;
-constexpr U8 FLAG_DISABLE_CONSTRAINT = 1 << 3;
-
-// target flags
-constexpr U8 FLAG_TARGET_POS = 1 << 4;
-constexpr U8 FLAG_TARGET_ROT = 1 << 5;
-constexpr U8 FLAG_HAS_DELEGATED = 1 << 6; // EXPERIMENTAL
-
-constexpr U8 MASK_POS = FLAG_TARGET_POS | FLAG_LOCAL_POS;
-constexpr U8 MASK_ROT = FLAG_TARGET_ROT | FLAG_LOCAL_ROT;
+// config flags
+constexpr U8 CONFIG_FLAG_LOCAL_POS = 1 << 0;
+constexpr U8 CONFIG_FLAG_LOCAL_ROT = 1 << 1;
+constexpr U8 CONFIG_FLAG_LOCAL_SCALE = 1 << 2;
+constexpr U8 CONFIG_FLAG_DISABLE_CONSTRAINT = 1 << 3;
+constexpr U8 CONFIG_FLAG_TARGET_POS = 1 << 4;
+constexpr U8 CONFIG_FLAG_TARGET_ROT = 1 << 5;
+constexpr U8 CONFIG_FLAG_HAS_DELEGATED = 1 << 6; // EXPERIMENTAL
```

Comment:
There are two sets of flags.  Those for properties set by the **LLIK::Joint::Config** class (above), and those that are the results of IK calculations (below).  Both sets of flags are combined to figure out what data needs to be harvested (see `LLIK::Joint::getHarvestFlags()` and where it is called).